src/main/groovy/wooga/gradle/unity/UnityTask.groovy

```diff
@@ -63,14 +64,16 @@ abstract class UnityTask extends DefaultTask
 
                 def unityPath = unityPath.get().asFile.absolutePath
                 def unityArgs = getAllArguments()
-
-                OutputStream outputStream = getOutputStream()
+                def unityEnvironment = environment.get()
+                def outputStream = getOutputStream()
+                environment.get().sort()
```

Comment:
what is the call to sort for? Is it in place?

Reply:
This wasn't meant to be committed (was checking functions); my apologies